### ISSUE TITLE
Rename-D365ComputerName -NewName failes due to MS SQL missing path

### ISSUE THREAD

**Issue by malioni91** [d365collaborative/d365fo.tools#646]
![image](https://user-images.githubusercontent.com/8109485/167261249-de14b449-5fe0-48c5-bec3-8575564c0ac0.png)

**Comment by Splaxi** [comment]
What version of d365fo are you running?

**Comment by malioni91** [comment]
> What version of d365fo are you running?

Hi. Thanks for the quick reply. I am running a VHD devbox of the following version: 
Installed product version : 10.0.24 (10.0.1084.80)
Installed platform version : Update48 (7.0.6253.76)

**Comment by Splaxi** [comment]
Was it downloaded as 10.0.24 or updated?

I suspect that we have some dependencies that was updated with latest release from Microsoft.

Please try to locate any other path (140,150,160,170) where the rsconfig.,exe is located on your machine.

**Comment by malioni91** [comment]
Hi, 

Its a freshly downloaded VHD from LCS. Actually I don't seem to find rsconfig.exe. Under 150 I see these. 

![image](https://user-images.githubusercontent.com/8109485/167295984-18cea16f-3514-4f29-9357-2e80a5911360.png)

**Comment by FH-Inway** [comment]
Also reported on Yammer: https://www.yammer.com/dynamicsaxfeedbackprograms/threads/1785310632681472

**Comment by FH-Inway** [comment]
As indicated in the Yammer thread, the rsconfig.exe tool is located in folder `C:\Program Files\Microsoft SQL Server Reporting Services\Shared Tools` on the new 10.0.24 VHD.
I tested [Rename-D365ComputerName](https://github.com/d365collaborative/d365fo.tools/blob/c34e0fd70311f7b368c618f0a54c29a917428d53/d365fo.tools/functions/rename-d365computername.ps1) with this folder path. However, during execution, I'm getting the following message:

> Report Server WMI Provider error:
> 
> Invalid namespace

So far, I was not able to find the cause of this message. However, I noticed that in the Report Server Configuration Manager, it already shows the SQL Server Name als `localhost`. So it seems this part of the cmdlet is not needed anymore.
For now, I will leave this part as is and just provide a fix for the folder.
